oam/install_scripts/mcs-exemgr.service.in

```diff
@@ -1,12 +1,11 @@
 [Unit]
 Description=mcs-exemgr
-PartOf=mcs-writeengineserver.service
-After=mcs-writeengineserver.service
+PartOf=mcs-primproc.service
+After=mcs-primproc.service
 
 [Service]
 Type=simple
-Environment="LD_PRELOAD=libjemalloc.so.1 libjemalloc.so.2"
-ExecStart=@ENGINE_BINDIR@/ExeMgr
+ExecStart=/usr/bin/env bash -c "LD_PRELOAD=$(ldconfig -p | grep -m1 libjemalloc | awk '{print $1}') exec @ENGINE_BINDIR@/ExeMgr"
```

Comment:
`$(ldconfig -p | grep -m1 libjemalloc | awk '{print $1}')` brings in the magic to get us correct .so (e.g., libjemalloc.so.1, libjemalloc.so.2). Double checked by searching for libjemalloc in output of  `lsof -p $(pgrep ExeMgr)`